code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling_subtypes.dm

```diff
@@ -3,23 +3,81 @@
 /// Will differentiate into the base giant spider (known colloquially as the "guard" spider).
 /mob/living/basic/spiderling/guard
 	grow_as = /mob/living/basic/giant_spider
+	name = "guard spiderling"
+	desc = "Furry and brown, it makes you shudder to look at it. This one has sparkling red eyes."
```

Comment:
If you are going to give these all individual descriptions, they shouldn't be copy/pasted from the adult spider because it does not make sense for them to have the same description.
Either write unique descs or don't have unique descs.

Reply:
This is why I have an issue with giant_spider being the exact same code and implementation as guard spider... well you see they have the same description because they are the same spider whoever coded the spiders decided to make one of the spiders from the spider infestation the guard spider the giant_spider I told maintainers about it and they said they didn't see this as an issue so I just ignored it

Reply:
This has absolutely no relevance to my review comment.